contributing/development/compiling/cross-compiling_for_ios_on_linux.rst

```diff
@@ -150,11 +114,11 @@ environment variable defined to anything.
 
 ::
 
-    $ export OSXCROSS_IOS=anything
+    export OSXCROSS_IOS="anything"
 
 Now you can compile for iPhone using SCons like the standard Godot
 way, with some additional arguments to provide the correct paths:
 
 ::
 
-    $ scons -j 4 platform=ios arch=arm64 target=template_release IOS_SDK_PATH="/path/to/iPhoneSDK" IOS_TOOLCHAIN_PATH="/path/to/iostoolchain" ios_triple="arm-apple-darwin11-"
+    scons platform=ios arch=arm64 target=template_release IOS_SDK_PATH="/path/to/iPhoneSDK" IOS_TOOLCHAIN_PATH="/path/to/iostoolchain" ios_triple="arm-apple-darwin11-"
```

Comment:
Is the iOS triple still up-to-date? Also, now that iOS is 64-bit only, could SCons default to a sensible value automatically so the user doesn't have to define it?

Reply:
It's the same value as used for official builds:

https://github.com/godotengine/godot-build-scripts/blob/d36a865ff1ced6e856628c3e2c82e01993bc56dc/build-ios/build.sh#L28-L31

It's only for osxcross, for building on macOS it should be empty, I guess it can be auto set based on arch when building with osxcross.